src/libutil/fs-sink.cc

```diff
@@ -38,6 +39,7 @@ void copyRecursive(
                 sink, to + "/" + name);
             break;
```

Comment:
We can also remove this `break` since it does nothing.

Reply:
nope the compiler is a big fool and can't figure that out unfortunately. i tried :)